includes/theme-help.php

```diff
@@ -346,162 +285,44 @@
-						Youtube.
+						Media (images, videos, and audio) can be uploaded to WordPress’s media library so that they are self-hosted. Generally, this is necessary for photos and video or audio that cannot be uploaded to an external service like Youtube.
 					</p>
 					<div class="well">
 						<p>
-							<strong>Note: if you need to add a Youtube, Vimeo, or other hosted video player, you do not
-							need to upload any files;</strong> simply paste the URL of the video into the editor content on its own line, and WordPress
-							will figure out the rest for you.
+							<strong>Note: if you need to add a Youtube, Vimeo, or other hosted video player, you do not need to upload any files;</strong> simply paste the URL of the video into the editor content on its own line, and WordPress will figure out the rest for you.
 						</p>
 					</div>
 					<img src="<?php echo THEME_HELP_IMG_URL?>/story-add-media.jpg" />
 					<p>
 						Media can be added to the post content by clicking on the Add Media button under the story title.
 					</p>
-					<img src="<?php echo THEME_HELP_IMG_URL?>/story-media-modal.jpg" />
 					<p>
-						In the window that appears, you can either browse existing media or upload a new file.  Click on a file’s thumbnail
-						to select it (if you just uploaded the file, it’ll be selected for you.)
+						In the window that appears, you can either browse existing media or upload a new file. Click on a file’s thumbnail to select it (if you just uploaded the file, it’ll be selected for you.)
 					</p>
 					<p>
-						The media’s details can then be modified on the right, under Attachment Details and Attachment Display Settings.
-						For images, you will need to add a caption (optional) and alt text.  The caption will be displayed below the image;
-						alt text will appear when you hover over the image, and provides additional information about the image to
-						screenreaders and search engines.  Adjust the image’s position under the Alignment dropdown and the size of the
-						image if necessary (note some images may not provide an image size option.)  Keep in mind that if no image position
-						is selected, the image will automatically be stretched to the full width of the post content column.
+						The media’s details can then be modified on the right, under Attachment Details and Attachment Display Settings. For images, you will need to add a caption (optional) and alt text. The caption will be displayed below the image; alt text will appear when you hover over the image, and provides additional information about the image to screenreaders and search engines. Adjust the image’s position under the Alignment dropdown and the size of the image if necessary (note some images may not provide an image size option.) Keep in mind that if no image position is selected, the image will automatically be stretched to the full width of the post content column.
```

Comment:
This sentence can be removed, since this behavior has been removed in v6:
>Keep in mind that if no image position is selected, the image will automatically be stretched to the full width of the post content column.